sass/components/_post.scss

```diff
@@ -2231,6 +2238,112 @@
     }
 }
 
+.edit-post__container {
```

Comment:
Are these names correct? This looks like it's for the `EditPost` components

Reply:
Yeah they are the correct names. However in order to eliminate the confusion I replaced `edit-post` with `edit-post-history`